yarn.lock

```diff
@@ -212,7 +212,7 @@
   resolved "https://registry.yarnpkg.com/@ungap/promise-all-settled/-/promise-all-settled-1.1.2.tgz#aa58042711d6e3275dd37dc597e5d31e8c290a44"
   integrity sha512-sL/cEvJWAnClXw0wHk85/2L0G6Sj8UB0Ctc1TEMbKSsmpRosqhwj9gWgFRZSrBr2f9tiXISwNhCPmlfqUqyb9Q==
 
-"@vscode/debugadapter-testsupport@^1.50.0":
+"@vscode/debugadapter-testsupport@^1.58.0-pre.0":
```

Comment:
This lock file looks invalid - it says to satisfy >= 1.58 it is using 1.51.0.

Did you manually update the lock file? You should rely on yarn to do that.

Reply:
I have updated it after run yarn command. Sorry for my mistake.